packages/picasso/src/Tooltip/useTooltipHandlers.ts

```diff
@@ -48,16 +48,16 @@ export const useTooltipHandlers = ({
       children
     }
   }
-  const handleClose = (event: ChangeEvent<{}>) => {
+  const handleClose = (event: SyntheticEvent | Event) => {
     onClose?.(event)
     closeTooltip()
   }
-  const handleOpen = (event: ChangeEvent<{}>) => {
+  const handleOpen = (event: SyntheticEvent | Event) => {
```

Comment:
```suggestion
  const handleOpen = (event: SyntheticEvent) => {
```
this should be enough 🤔